frontend/scss/components/templates/roadmap.scss

```diff
@@ -412,27 +420,43 @@
             color: #000;
             background: color('gallery');
             font-size: .9em;
-            white-space: nowrap;
+          }
+
+          &:after {
+            content: '';
+            position: absolute;
+            bottom: 0;
+            width: 100%;
+            height: 50px;
+            background: linear-gradient(180deg,hsla(0,0%,100%,0),#fff 100%);
```

Comment:
New kids on the block with `hsla` or what? 😉 I'd favor staying consistent. Ideally, this would be:

```css
linear-gradient(180deg, transparentize(color('white'), 1), color('white') 100%);
```